### DISCUSSION TITLE
Add feature to exclude specific folder/files in Vault QA

### DISCUSSION THREAD

**Discussion by sulth1** [logancyang/obsidian-copilot#346] [Ideas]
Hello, 

As the title mention, it would help to be able to exclude specific folders from being indexed/used as context. Smart Connections has implemented that. 
Thank you so much for the plugin, again!

**Comment by logancyang** [comment]
Code has been merged, will be in the next release!

**Reply by sulth1** [reply]
Absolutely awesone, thank you! 

However, I tried excluding multiples folder, and it doesn't seem to be working. Maybe I am doing something wrong?
After refreshing the indexing, and asking something in the vault QA, is still links to the other folders bar the first one. 

![image](https://github.com/logancyang/obsidian-copilot/assets/156309844/f48ca33a-1e9b-4c87-9d1f-b50d74c3d40f)

**EDIT : Nevermind, I needed to clear local vector stores and reindex everything. Works wonderful now!**

**Reply by logancyang** [reply]
@sulth1 Glad it works now!
